core/sql/optimizer/SearchKey.h

```diff
@@ -1042,6 +1045,10 @@ class HbaseSearchKey : public SearchKey
    ValueIdSet requiredOutputColumns_;
 
    ValueIdSet nonKeyPreds_;
+
+   // is a constant folded pred that returns FALSE always. SK has 
+   // min/max for begin/end reversed.
+   NABoolean isFalsePred_;
```

Comment:
In the longer term it would be nice to make this part of the generic SearchKey, not just HBaseSearchKey.